Solutions/ApacheHTTPServer/Data/Solution_Apache Http Server.json

```diff
@@ -1,8 +1,8 @@
 {
   "Name": "ApacheHTTPServer",
-  "Author": "Manoj Arimanda - v-marimanda@microsoft.com",
-  "Description": "The Apache HTTP Server data connector provides the capability to ingest [Apache HTTP Server](http://httpd.apache.org/) events into Azure Sentinel. Refer to [Apache Logs documentation](https://httpd.apache.org/docs/2.4/logs.html) for more information.",
-  "WorkbookDescription": "Sets the time name for analysis",
+  "Author": "Microsoft - support@microsoft.com",
+  "Logo": "<img src=\"https://raw.githubusercontent.com/Azure/Azure-Sentinel/master/Solutions/ApacheHTTPServer/Workbooks/Images/Logo/apache.svg\" width=\"75px\" height=\"75px\">",
```

Comment:
Please remove the forward slash i.e. `\` from the URL. If we try to open this url `https://raw.githubusercontent.com/Azure/Azure-Sentinel/master/Solutions/ApacheHTTPServer/Workbooks/Images/Logo/apache.svg\` with forward slash it gives `400: Invalid request` error.